### DISCUSSION TITLE
Elfeed needs a logo or icon

### DISCUSSION THREAD

**Discussion by alphapapa** [emacs-elfeed/elfeed#600] [General]
Hey Chris,

I've started using Elfeed in a dedicated frame with frame-purpose, and now I need an icon for that frame's X windows.  :)  Do you have any ideas, or would you accept suggestions?

Thanks.

**Comment by skeeto** [comment]
I've always imagined the mascot being some sort of elf, like in the 
style of Keebler or Santa Claus. I'm not wedded to that idea, though. 
Did you have something specific in mind?

**Comment by alphapapa** [comment]
Haha, elf, that's clever.  No, I don't have anything in mind.  I can put simple things together in Inkscape or Gimp, but I'm not an artist, so I won't be drawing any original elves for a logo.  ;)  Maybe we could find some free-license art to use though...

**Comment by alphapapa** [comment]
Well, a quick look on Google Images for freely licensed "elf" images does not look promising, haha.

**Comment by emacsomancer** [comment]
An elf with a feed-bag strapped to his face (where the feedbag has 'RSS' printed on it).

**Comment by minad** [comment]
Converting to discussion. If someone with artistic talent and creativity comes up with a nice logo it would be welcome, I guess.

**Comment by paniash** [comment]
<img width="1541" height="1536" alt="elfeed-elf-cropped" src="https://github.com/user-attachments/assets/b1ecf6e0-2113-4ec8-993f-468e482f71a4" />

@minad Hi, here's a first draft. Let me know if you think some changes are needed. (Full disclosure: I did use AI to make some modifications).

**Reply by minad** [reply]
Thanks. I think the inner circle only is nice. Maybe it could be a little bit simpler with fewer gradients (e.g. the glow of the book)? I am not sure about the book, but I don't really have a better idea.

**Reply by paniash** [reply]
So then we forgo the outer circle and the "elfeed" text? I think the book is nice to show that people are technically "reading" feeds as a more textual approach to consuming online content. I thought the glow might be nice but if you think otherwise, I can have that removed as well.

**Reply by minad** [reply]
Yes, I would avoid the outer text. We could use the inner circle as logo and Elfeed would be written next to it anyway in the README. I am just suggesting to try out different versions. Let's also see what others have to say.

**Reply by minad** [reply]
One could also write lfeed next to the round E with the elf, but maybe you've already tried this out and readability wasn't good? I am all for a relatively simple logo without too many elements, like the inner circle only.

**Reply by minad** [reply]
I should add that I like your proposal overall, so please don't take my thoughts as criticism. I like that it has a friendly elf, and it integrates the E (for Elfeed and Emacs) and RSS neatly.

**Reply by minad** [reply]
Maybe instead of a book one could use some papers or a papyrus roll, since feeds are also endless ;)

**Reply by paniash** [reply]
<img width="918" height="894" alt="elfeed-elf-outer-circle-removed" src="https://github.com/user-attachments/assets/f7aefda2-5336-48ef-aa14-862478a89f69" />


Thanks! Your suggestions do make good sense. For now, I've removed the text and outer circle. I like your idea about writing the lfeed part next to the round E (I'll make those changes soon). Happy to hear from the other maintainers about their thoughts too.

**Comment by yantar92** [comment]
Some brainstorming (with LLM)
<img width="1912" height="3866" alt="image" src="https://github.com/user-attachments/assets/e7f640ee-aeea-4789-85d1-f8e17f02c6d9" />

**Reply by yantar92** [reply]
And a couple more, themed after the nearest book on my table. We cannot use all these directly, but if anything catches the eye, we can try making a proper logo with the elements and design that are liked by all.
<img width="1368" height="1032" alt="image" src="https://github.com/user-attachments/assets/64d6980d-0351-484d-9e1b-3a0301aa06cb" />
